build/GitCommitInfo.targets

```diff
@@ -23,11 +19,20 @@
     &lt;/PropertyGroup&gt;
 &lt;/Project&gt;
       </GitCommitInfoPropsContent>
-    </PropertyGroup>
 
-    <WriteLinesToFile File="$(GitCommitInfoProps)"
-                      Lines="$(GitCommitInfoPropsContent)"
-                      Condition=" '$(ShouldOverwriteThePropsFile)' == 'true' "
-                      Overwrite="true" />
+      <ExistingGitCommitInfoProps Condition=" Exists('$(GitCommitInfoProps)') ">
+        $([System.IO.File]::ReadAllText($(GitCommitInfoProps)))
+      </ExistingGitCommitInfoProps>
+
+      <ShouldOverwriteGitCommitInfoPropsFile>false</ShouldOverwriteGitCommitInfoPropsFile>
+      <ShouldOverwriteGitCommitInfoPropsFile
+        Condition=" !Exists('$(GitCommitInfoProps)') Or
```

Comment:
Do you really need the exists condition? If it does not exist, won't this be different than GitCommitInfoPropsContent?